daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/Daml3ScriptTestRunner.scala

```diff
@@ -33,12 +40,12 @@ class DamlScriptTestRunner extends AnyWordSpec with CantonFixture with Matchers
           |ScriptExample:initializeUser SUCCESS
           |ScriptExample:test SUCCESS
           |ScriptTest:clearUsers SUCCESS
-          |ScriptTest:failingTest FAILURE (com.daml.lf.engine.script.Script$FailedCmd: Command submit failed: FAILED_PRECONDITION: UNHANDLED_EXCEPTION(9,XXXXXXXX): Interpretation error: Error: Unhandled Daml exception: DA.Exception.AssertionFailed:AssertionFailed@3f4deaf1{ message = "Assertion failed" }. Details: Last location: [DA.Internal.Exception:168], partial transaction: ...
+          |ScriptTest:failingTest FAILURE (com.daml.lf.engine.script.Script$FailedCmd: Command submit failed: FAILED_PRECONDITION: UNHANDLED_EXCEPTION(9,XXXXXXXX): Interpretation error: Error: Unhandled Daml exception: DA.Exception.AssertionFailed:AssertionFailed@XXXXXXXX{ message = "Assertion failed" }. Details: Last location: [DA.Internal.Exception:168], partial transaction: ...
```

Comment:
Do we want to change XXXXXXXX back to some correct hash?

Reply:
Ah, I can return them, though I'm not sure if we should be relying on them being static, doesn't seem like important information to this test